compiler/damlc/tests/daml-test-files/Choice_no_params_template.daml

```diff
@@ -22,8 +22,8 @@ template Bond
       Split : (ContractId Bond, ContractId Bond)
         with splitAmount : Decimal
         do
-          f <- create $ this with amount = splitAmount
-          s <- create $ this with amount = amount - splitAmount
+          f <- create this with amount = splitAmount
```

Comment:
Does this change require removing the `$`s or is this an unrelated change because they are redundant?

Reply:
This is unrelated, it just so happened I found them while testing. They are redundant.